style.css

```diff
@@ -310,3 +345,64 @@ body {
   border-radius: 8px;
   color: #fff;
 }
+
+@media only screen and (min-width: 708px) {
+  .work {
+    display: flex;
+    max-width: 70%;
+    margin: auto;
+    justify-content: space-between;
+  }
+
+  header {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  body {
+    width: 100%;
+    margin: 0;
+    padding: 0%;
+  }
+
+  .desktop {
+    margin-left: 50px;
+  }
+
+  .about-section {
+    display: flex;
+    width: 100%;
+  }
+
+  .about-left {
+    max-width: 40%;
+  }
+
+  .about-right {
+    flex-grow: 2;
+    margin-left: 50px;
+  }
+
+  main {
+    padding: 0%;
+    margin: 0%;
+  }
+
+  #nav-link {
+    display: flex;
+  }
+
+  nav > svg {
+    display: none;
+  }
+
+  #nav-link li {
+    padding-right: 20px;
+    list-style: none;
+  }
+
+  #nav-link li a {
+    text-decoration: none;
+    color: #344563;
+  }
+}
```

Comment:
- [ ] Please, as required in the project requirements, you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided.
- your design:
<img width="704" alt="image" src="https://user-images.githubusercontent.com/77190203/160138856-1ed17527-84a2-4acf-be6b-20284ea2ffd9.png">

-  Figma design:
<img width="431" alt="image" src="https://user-images.githubusercontent.com/77190203/160139157-cce45789-b9f8-46a8-9b14-1e38fed24c62.png">